### ISSUE TITLE
Calling `DidYouMean::SPELL_CHECKERS.....`

### ISSUE THREAD

**Issue by Tewboo** [ralixjs/rails-ralix-tailwind#13]
[5:10][ruby-3.2.1]jesse:~/rails-ralix-tailwind@main % bin/setup  
== Installing dependencies ==
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
The Gemfile's dependencies are satisfied
------------------------
Above is the warning I received, what is the meaning of this, can I disappear from this warning?
Thanks in advance.

Other Info:
Ubuntu 2004
Rails 7
Ruby 3.2.1

**Comment by markets** [comment]
hey @Tewboo 👋🏼 

That's just a warning, no worries, I'll try to solve it in future iterations. Thanks for reporting!

**Comment by markets** [comment]
Fixed in https://github.com/ralixjs/rails-ralix-tailwind/commit/104e3c65ec6d7cfada781fdd906fe8b7e641fac3 (by upgrading Bundler)

**Comment by Tewboo** [comment]
> hey @Tewboo 👋🏼
> 
> That's just a warning, no worries, I'll try to solve it in future iterations. Thanks for reporting!

Thanks, got it, I am so appreciate to receive your reply, 
And thanks again for sharing this wonderful framwork for us to use;

**Comment by Tewboo** [comment]
> Fixed in [104e3c6](https://github.com/ralixjs/rails-ralix-tailwind/commit/104e3c65ec6d7cfada781fdd906fe8b7e641fac3) (by upgrading Bundler)

Maybe a easy question, I checked the gemfile and package file, I guess what I should do is change to version number and run `bundle` right, 
So sorry to ask this easy question, I am really newbie for this rails tech.
Thanks in advance.

Below is what I am going to revise my code:
```
gem "puma", "~> 6.2"
"tailwindcss": "^3.3.1",
```

**Comment by jun-jing** [comment]
still facing this issue after `bundle install` and `yarn` to update the dependency, still trying to figure it out. below is what I did, thansk for checking:

What output it was when run `bin/dev`:
```
[15:32][ruby-3.2.1]jesse:~/Documents/Rails_Ralix_ForChatgptProject@master✗✗✗ % bin/dev
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
15:32:20 web.1  | started with pid 66589
15:32:20 js.1   | started with pid 66590
15:32:20 css.1  | started with pid 66591
15:32:20 web.1  | Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
15:32:20 css.1  | yarn run v1.22.19
15:32:20 js.1   | yarn run v1.22.19
15:32:20 css.1  | $ tailwindcss -i ./app/assets/stylesheets/application.css -o ./app/assets/builds/application.css --minify --watch
15:32:21 js.1   | $ esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=assets --watch
15:32:21 js.1   | [watch] build finished, watching for changes...
15:32:22 css.1  | 
15:32:22 css.1  | Rebuilding...
15:32:23 web.1  | => Booting Puma
15:32:23 web.1  | => Rails 7.0.4.2 application starting in development 
15:32:23 web.1  | => Run `bin/rails server --help` for more startup options
15:32:24 css.1  | 
15:32:24 css.1  | Done in 1744ms.
15:32:24 web.1  | Puma starting in single mode...
15:32:24 web.1  | * Puma version: 6.2.2 (ruby 3.2.1-p31) ("Speaking of Now")
15:32:24 web.1  | *  Min threads: 5
15:32:24 web.1  | *  Max threads: 5
15:32:24 web.1  | *  Environment: development
15:32:24 web.1  | *          PID: 66589
15:32:24 web.1  | * Listening on http://127.0.0.1:3000
15:32:24 web.1  | * Listening on http://[::1]:3000
15:32:24 web.1  | Use Ctrl-C to stop
```

What I did to `package.json` and `Gemfile`:
1, `gem "puma", "~> 6.2"`
2, `"tailwindcss": "^3.3.1",`
